src/sidebar/store/modules/test/viewer-test.js

```diff
@@ -1,22 +1,33 @@
 'use strict';
 
 const viewer = require('../viewer');
+const createStore = require('../../create-store');
 
-const util = require('../../util');
+let store;
```

Comment:
Usually we put `let` variables that are initialized in `beforeEach` blocks at the top of the nearest `describe` block.